Homework/Van_Le/lesson3/KClosetPointsToOrigin/KClosetPointsToOrigin.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Linq;
+
+namespace KClosetPointsToOrigin
+{
+    public class KClosetPointsToOrigin
+    {
+        public int[][] KClosest(int[][] points, int k)
+        {
+            return points.OrderBy(p => Math.Sqrt(p[0] * p[0] + p[1] * p[1])).Take(k).ToArray();
```

Comment:
lmao, interviewer will definitely ask you not using built-in func. Let's try impl quick-select